index.html

```diff
@@ -99,12 +99,16 @@
         </li>
         <li class="point speaker">
           <time>8:00p</time>
-          <!--
-            To add yourself to this slot, change the following two lines
-            and add a short description paragraph in the pull request.
-          -->
-          <small>Listen to our first speaker, or better yet</small>
-          <div><a href="https://github.com/corridorjs/corridorjs.github.io/edit/master/index.html">BE OUR FIRST SPEAKER</a></div>        </li>
+          <small>Is Front-End dev the new Back-End dev?</small>
+          <div>
+            The destinction between fron-end and back-end development style has changed recently. These days font-end dev
+            looks and feel a lot like node.js dev. In this short talk, we'll explore that as well as some of the
+            things we must now worrie about in this new world.
```

Comment:
I know this is a closed PR *and* this is a trivial note, but "worrie" -> "worry"